tests/helper/helper.go

```diff
@@ -432,3 +449,37 @@ func RemoveANSI(input string) string {
 	reg := regexp.MustCompile(`(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]`)
 	return reg.ReplaceAllString(input, "")
 }
+
+func FindPodLogs(t *testing.T, kc *kubernetes.Clientset, namespace, label string) string {
+	var podLogs string
+	t.Logf("Searching for pod logs.........")
+	pods, err := kc.CoreV1().Pods(namespace).List(context.TODO(),
+		metav1.ListOptions{LabelSelector: label})
+	if err != nil {
+		assert.NoErrorf(t, err, "no pod in the list - %s", err)
+	}
+	var podLogRequest *rest.Request
+	for _, v := range pods.Items {
+		podLogRequest = kc.CoreV1().Pods(namespace).GetLogs(v.Name, &corev1.PodLogOptions{})
+	}
```

Comment:
Are we assuming here only one pod matches the label (considering we overwrite the podLogRequest)?

Reply:
> Are we assuming here only one pod matches the label (considering we overwrite the podLogRequest)?

hi @v-shenoy , yes , the function was made as per influxdb scaler & then transferred to helper so might have missed this detail. what do you suggest in this case ?

Reply:
Maybe move all the below lines into the loop, and we return a list of logs (although we don't have much use for it right now, but the function will be more general at least).

Reply:
that's what i am working on right now. or we can just list out name of pods because not many tests are trying to access the logs but a function for list of pods would be much more useful
What Do you think?

Reply:
Could you elaborate what you mean by list out name of pods?

Reply:
Hi @v-shenoy , generalised the function for logs in latest code ,as for the last comment : If the function were to return only a list of podnames then the scaler test would try finding whatever information regarding the pod using the particular pod name. It depends on scaler test demands if more tests demand pod details and not logs then we can keep it till the podList is given. 

For time being i made the function generalised , thanks for the suggestion!